### PR TITLE
Partial support for Fuel Bytes type

### PR DESCRIPTION
**Pull request by DZakh**
Treat it as unknown for now

### CODE REVIEW

File: scenarios/fuel_test/src/AllEventsHandlers.ts [hunk 1]
```diff
@@ -196,7 +198,15 @@ AllEvents.Option2.handler(async ({ event }) => {
...
-AllEvents.Vec.handler(async ({ event }) => {
+AllEvents.VecLog.handler(async ({ event }) => {
   vecLogSchema.assert(event.params)!;
   expectType<AssertSchemaType<typeof event.params, typeof vecLogSchema>>(true);
 });
+
+const bytesLogSchema = S.unknown;
+AllEvents.BytesLog.handler(async ({ event }) => {
+  bytesLogSchema.assert(event.params)!;
+  expectType<AssertSchemaType<typeof event.params, typeof bytesLogSchema>>(
```
Comment (JasoonS) [inline scenarios/fuel_test/src/AllEventsHandlers.ts:209]: I think I need a little bit more explanation on this. So it's seen as an unknown type.

Does that mean it's the data exactly raw as is from the chain and it's up to the user to parse this type into whatever they want?

**Review by JasoonS — APPROVED** [review]
If I understand this change correctly, it doesn't change any functionality. It just explicitly marks the type of the data to unknown. And then the user can do whatever they want with that data, but we don't pause it for them.

File: scenarios/fuel_test/src/AllEventsHandlers.ts (hunk 1 above)
Comment (DZakh) [reply scenarios/fuel_test/src/AllEventsHandlers.ts:209]: > Does that mean it's the data exactly raw as is from the chain and it's up to the user to parse this type into whatever they want?

Yes

**Comment by DZakh** [comment]
> If I understand this change correctly, it doesn't change any functionality. It just explicitly marks the type of the data to unknown. And then the user can do whatever they want with that data, but we don't pause it for them.

Yes, previously it was treated as an object with some weird fields, which wasn't correct and broke codegen for schemas. In this version, we still don't have an exact type for bytes, but at least it's not lying.

**Pull request merged**